dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/Standard/HVDCStandard_INIT.mo

```diff
@@ -0,0 +1,76 @@
+
+protected
+
+  Types.ComplexVoltagePu u10Pu "Start value of complex voltage at terminal 1 in p.u (base UNom)";
+  Types.ComplexApparentPowerPu s10Pu "Start value of complex apparent power at terminal 1 in p.u (base SnRef) (receptor convention)";
+  flow Types.ComplexCurrentPu i10Pu "Start value of complex current at terminal 1 in p.u (base UNom, SnRef) (receptor convention)";
+  Types.ComplexVoltagePu u20Pu "Start value of complex voltage at terminal 2 in p.u (base UNom)";
+  Types.ComplexApparentPowerPu s20Pu "Start value of complex apparent power at terminal 2 in p.u (base SnRef) (receptor convention)";
+  flow Types.ComplexCurrentPu i20Pu "Start value of complex current at terminal 2 in p.u (base UNom, SnRef) (receptor convention)";
+  Types.PerUnit Ip10Pu "Start value of active current at terminal 1 in p.u (base SNom)";
+  Types.PerUnit Iq10Pu "Start value of reactive current at terminal 1 in p.u (base SNom)";
+  Types.PerUnit Ip20Pu "Start value of active current at terminal 2 in p.u (base SNom)";
+  Types.PerUnit Iq20Pu "Start value of reactive current at terminal 2 in p.u (base SNom)";
+  Types.PerUnit Udc10Pu "Start value of dc voltage at terminal 1 in p.u (base SNom, UNom)";
+  Types.PerUnit Udc20Pu "Start value of dc voltage at terminal 2 in p.u (base SNom, UNom)";
+  Types.VoltageModulePu URef10Pu "Start value of the voltage reference for the side 1 of the HVDC link in p.u (base UNom)";
+  Types.VoltageModulePu URef20Pu "Start value of the voltage reference for the side 1 of the HVDC link in p.u (base UNom)";
+  Types.ReactivePowerPu QRef10Pu "Start value of reactive power reference at terminal 1 in p.u (base SNom) (generator convention)";
+  Types.ReactivePowerPu QRef20Pu "Start value of reactive power reference at terminal 2 in p.u (base SNom) (generator convention)";
+  Types.ReactivePowerPu PRef0Pu "Start value of reactive power reference in p.u (base SNom) (generator convention)";
+  Real modeU10 "Start value of the real assessing the mode of the control: 1 if U mode, 0 if Q mode";
+  Real modeU20 "Start value of the real assessing the mode of the control: 1 if U mode, 0 if Q mode";
+
+equation
+
+  s10Pu = Complex(P10Pu, Q10Pu);
+  u10Pu = ComplexMath.fromPolar(U10Pu, UPhase10);
+  s10Pu = u10Pu * ComplexMath.conj(i10Pu);
+  s20Pu = Complex(P20Pu, Q20Pu);
+  u20Pu = ComplexMath.fromPolar(U20Pu, UPhase20);
+  s20Pu = u20Pu * ComplexMath.conj(i20Pu);
+  P10Pu = - U10Pu * Ip10Pu * (SNom/SystemBase.SnRef);
+  Q10Pu = U10Pu * Iq10Pu * (SNom/SystemBase.SnRef);
+  P20Pu = - U20Pu * Ip20Pu * (SNom/SystemBase.SnRef);
+  Q20Pu = U20Pu * Iq20Pu * (SNom/SystemBase.SnRef);
+  Udc10Pu = 1;
+  Udc20Pu = 1 + RdcPu * PRef0Pu;
+  QRef10Pu = - Q10Pu / (SNom/SystemBase.SnRef);
+  QRef20Pu = - Q20Pu / (SNom/SystemBase.SnRef);
+  PRef0Pu = - P10Pu / (SNom/SystemBase.SnRef);
+  URef10Pu = U10Pu - Lambda * Q10Pu / (SNom/SystemBase.SnRef);
+  URef20Pu = U20Pu - Lambda * Q20Pu / (SNom/SystemBase.SnRef);
+  modeU10 = 1;
+  modeU20 = 1;
```

Comment:
So I think you shouldn't initialize it here but perhaps refer to an external parameter. What do you think? We can discuss it tomorrow for example.

Reply:
Fixed